<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileDialog</class>
 <widget class="QDialog" name="FileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="2">
    <widget class="QPushButton" name="pushButton_selectDevice">
     <property name="text">
      <string>Selection périphérique fichier</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="QTextEdit" name="textEditRponse"/>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="pushButton_sendFile">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Send file</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="5">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Programme test d'envoie de trame ou de fichier</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="pushButton_Quit">
     <property name="text">
      <string>Quitter</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pushButton_SelectFile">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Select file</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEditTrame"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_Quit</sender>
   <signal>clicked()</signal>
   <receiver>FileDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>608</x>
     <y>447</y>
    </hint>
    <hint type="destinationlabel">
     <x>674</x>
     <y>451</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
